### DISCUSSION TITLE
Bot Up 24/7?

### DISCUSSION THREAD

**Discussion by ZaibatsuX** [eritislami/evobot#1223] [Q&A]
Hello, How can i get the bot to up 24/7? i really need it, thanks.

**Comment by wolf-yuan-6115** [comment]
You can deploy EvoBot to Heroku, Replit, or any other hosting platforms you want.

**Reply by ZaibatsuX** [reply]
i cant with replit it need a subscription

**Comment by wolf-yuan-6115** [comment]
Then use heroku

**Reply by ZaibatsuX** [reply]
how??
